### ISSUE TITLE
Review Namada Ledger app transaction formatting

### ISSUE THREAD

**Issue by murisi** [anoma/namada#3500]
Could we review the following details pertaining to the Namada Ledger app test vectors generated by #3494?:
* Transaction coverage: are all the desired transaction types generated by the test vector generator?
* Variation coverage: are all possible variations (enum variants) of each transaction generated by the test vector generator?
* Display coverage: are all fields of a given transaction displayed in expert mode without any omission?
* User-friendliness of the normal mode:
  * Is too much information being displayed for the average user?
  * Are the relevant details of each transaction being printed in normal mode?
  * Are the outputs in normal mode being ordered sensibly?
  * Are there better ways of grouping/organizing the normal mode outputs? (Especially for `Transfer`s)

**Comment by cwgoes** [comment]
@murisi Is this list of review instructions still up-to-date?

**Comment by murisi** [comment]
> @murisi Is this list of review instructions still up-to-date?

@cwgoes Yes the instructions are still up to date, but the attachments are now out of date. See attached the latest test vectors. Also, please focus on the `output` and `output_expert` fields because these represent the lines that users will actually see on the hardware wallet as they press the left and right buttons. Thanks!

[schema43.txt](https://github.com/user-attachments/files/16802284/schema43.txt)
[vectors43.json](https://github.com/user-attachments/files/16802320/vectors43.json)
[debugs43.txt.gz](https://github.com/user-attachments/files/16802361/debugs43.txt.gz)
